mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1183,14 +1191,42 @@ trait Completions { this: MetalsGlobal =>
       override def contribute: List[Member] = {
         if (start < 0) Nil
         else {
-          typed.tpe.members.iterator
+
+          val overrideMembers: List[OverrideDefMember] =
+            typed.tpe.members.iterator
+              .filter(isOverridableMethod)
+              .map(OverrideCandidate.apply)
+              .filter { candidate =>
+                CompletionFuzzy.matchesSubCharacters(
+                  prefix,
+                  candidate.filterText
+                )
+              }
+              .map(_.toMember)
+              .toList
+
+          val allAbstractMembers = typed.tpe.members.iterator
             .filter(isOverridableMethod)
             .map(OverrideCandidate.apply)
-            .filter { candidate =>
-              CompletionFuzzy.matchesSubCharacters(prefix, candidate.filterText)
-            }
             .map(_.toMember)
+            .filter(_.sym.isAbstract)
             .toList
+            .map(_.edit.getNewText)
+
+          if (allAbstractMembers.length > 1 && overrideMembers.length > 1) {
+            val implementAll: TextEditMember = new TextEditMember(
+              prefix,
+              new l.TextEdit(
+                range,
+                allAbstractMembers.reverse.mkString("\n")
+              ),
+              completionsSymbol("implement"),
+              label = Some("Implement all members"),
+              detail = Some(s"(${allAbstractMembers.length} members)")
```

Comment:
"all members (3 members)" feels a bit redundant
```suggestion
              detail = Some(s" (${allAbstractMembers.length} total)")
```